pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[tool.poetry]
+name = "openadapt"
+version = "0.1.0"
+description = "GUI Process Automation with Transformers"
+authors = ["Richard Abrich <richard.abrich@mldsai.com>"]
+readme = ["README.md", "LICENSE"]
+
+repository = "https://github.com/mldsai/openadapt"
+homepage = "https://openadapt.ai/"
+
+
+[tool.poetry.dependencies]
+python = "3.10.x"
+atomacos = {git = "https://github.com/abrichr/atomacos.git", markers = "sys_platform == 'darwin'"}
+pynput = {git = "https://github.com/abrichr/pynput.git"}
```

Comment:
Any way to specify these as editable? Possibly relevant: https://stackoverflow.com/a/69711730/95989